keras_preprocessing/image.py

```diff
@@ -1257,7 +1252,7 @@ def fit(self, x,
             self.principal_components = (u * s_inv).dot(u.T)
 
 
-class Iterator(keras_utils.Sequence):
+class Iterator(object):
```

Comment:
This is a breaking change right?

Reply:
Sorry, I'm late in seeing this. This is *not* a breaking change. `keras_preprocessing.Iterator` is not meant to be used as-is, it is meant to be redefined by client packages (like Keras and tf.keras) via dual inheritance, i.e. `keras.preprocessing.Iterator` = `keras.Sequence` + `keras_preprocessing.Iterator`.